Code/GraphMol/MolDraw2D/Wrap/testMolDraw2DQt.py

```diff
@@ -0,0 +1,74 @@
+from rdkit import RDConfig
+import unittest
+from rdkit import Chem
+from rdkit.Chem import Draw, AllChem, rdDepictor
+from rdkit.Chem.Draw import rdMolDraw2D
+
+import sys
+from PyQt5.Qt import *
+import sip
+
+
+class TestCase(unittest.TestCase):
+
+  def setUp(self):
+    pass
+
+  def testSIPBasics(self):
+    m = Chem.MolFromSmiles('c1ccccc1O')
+    Draw.PrepareMolForDrawing(m)
+    qimg = QImage(250, 200, QImage.Format_RGB32)
+    qptr = QPainter(qimg)
+    qptr._img = qimg
```

Comment:
what does this line do?

Reply:
lifetime management for qimg: it ensures that qimg won't be deleted while qptr is still alive